manifests/m/MisfitCode/MisfitModel3D/1.2.4/MisfitCode.MisfitModel3D.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MisfitCode.MisfitModel3D
+PackageVersion: 1.2.4
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://deac-ams.dl.sourceforge.net/project/misfitmodel3d/misfitmodel3d-stable/mm3d-1.2.4/mm3d-1_2_4-win32-installer.exe

```

Comment:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/misfitmodel3d/files/misfitmodel3d-stable/mm3d-1.2.4/mm3d-1_2_4-win32-installer.exe/download
```

Reply:
still gives Validation unapproved URL.

Reply:
PR will need to be reviewed by an administrator to add the `Validation-Completed` label.